src/evidently/types.ts

```diff
@@ -0,0 +1,54 @@
+export type EvaluationCallback = (err?: Error, res?: EvaluationResult) => void;
+
+export type ContextType = { [k: string]: any };
+export type PartialEvidentlyConfig = {
+    project: string;
+    endpoint?: string;
+};
+export type EvidentlyConfig = {
+    project?: string;
+    endpoint: string;
+    endpointUrl: URL;
+};
+export type InitializeFeaturesRequest = {
+    features: string[];
+    entityId?: string;
+    context?: ContextType;
+};
+type VariationValue = {
+    boolValue?: boolean;
+    doubleValue?: number;
+    longValue?: number;
+    stringValue?: string;
+};
+export type EvaluationResult = {
+    feature: string;
+    reason: string;
+    value: VariationValue;
+    variation: string;
+};
+export type EvaluationResults = {
+    [feature: string]: EvaluationResult;
+};
+
+export type EvaluationRequest = {
+    entityId: string;
+    evaluationContext?: string;
+    feature: string;
+};
+export type BatchEvaluateFeatureRequest = {
+    requests: EvaluationRequest[];
+};
+
+export type BatchEvaluationResult = {
```

Comment:
nit: `EvaluationResult`

Reply:
This is actually the type that is returned from the API, we're still making BatchEvaluateFeature requests to the API